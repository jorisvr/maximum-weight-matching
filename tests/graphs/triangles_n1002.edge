p edge 1002 1335
e 1 2 1
e 1 3 1
e 2 3 1
e 4 5 1
e 4 6 1
e 5 6 1
e 7 8 1
e 7 9 1
e 8 9 1
e 10 11 1
e 10 12 1
e 11 12 1
e 13 14 1
e 13 15 1
e 14 15 1
e 16 17 1
e 16 18 1
e 17 18 1
e 19 20 1
e 19 21 1
e 20 21 1
e 22 23 1
e 22 24 1
e 23 24 1
e 25 26 1
e 25 27 1
e 26 27 1
e 28 29 1
e 28 30 1
e 29 30 1
e 31 32 1
e 31 33 1
e 32 33 1
e 34 35 1
e 34 36 1
e 35 36 1
e 37 38 1
e 37 39 1
e 38 39 1
e 40 41 1
e 40 42 1
e 41 42 1
e 43 44 1
e 43 45 1
e 44 45 1
e 46 47 1
e 46 48 1
e 47 48 1
e 49 50 1
e 49 51 1
e 50 51 1
e 52 53 1
e 52 54 1
e 53 54 1
e 55 56 1
e 55 57 1
e 56 57 1
e 58 59 1
e 58 60 1
e 59 60 1
e 61 62 1
e 61 63 1
e 62 63 1
e 64 65 1
e 64 66 1
e 65 66 1
e 67 68 1
e 67 69 1
e 68 69 1
e 70 71 1
e 70 72 1
e 71 72 1
e 73 74 1
e 73 75 1
e 74 75 1
e 76 77 1
e 76 78 1
e 77 78 1
e 79 80 1
e 79 81 1
e 80 81 1
e 82 83 1
e 82 84 1
e 83 84 1
e 85 86 1
e 85 87 1
e 86 87 1
e 88 89 1
e 88 90 1
e 89 90 1
e 91 92 1
e 91 93 1
e 92 93 1
e 94 95 1
e 94 96 1
e 95 96 1
e 97 98 1
e 97 99 1
e 98 99 1
e 100 101 1
e 100 102 1
e 101 102 1
e 103 104 1
e 103 105 1
e 104 105 1
e 106 107 1
e 106 108 1
e 107 108 1
e 109 110 1
e 109 111 1
e 110 111 1
e 112 113 1
e 112 114 1
e 113 114 1
e 115 116 1
e 115 117 1
e 116 117 1
e 118 119 1
e 118 120 1
e 119 120 1
e 121 122 1
e 121 123 1
e 122 123 1
e 124 125 1
e 124 126 1
e 125 126 1
e 127 128 1
e 127 129 1
e 128 129 1
e 130 131 1
e 130 132 1
e 131 132 1
e 133 134 1
e 133 135 1
e 134 135 1
e 136 137 1
e 136 138 1
e 137 138 1
e 139 140 1
e 139 141 1
e 140 141 1
e 142 143 1
e 142 144 1
e 143 144 1
e 145 146 1
e 145 147 1
e 146 147 1
e 148 149 1
e 148 150 1
e 149 150 1
e 151 152 1
e 151 153 1
e 152 153 1
e 154 155 1
e 154 156 1
e 155 156 1
e 157 158 1
e 157 159 1
e 158 159 1
e 160 161 1
e 160 162 1
e 161 162 1
e 163 164 1
e 163 165 1
e 164 165 1
e 166 167 1
e 166 168 1
e 167 168 1
e 169 170 1
e 169 171 1
e 170 171 1
e 172 173 1
e 172 174 1
e 173 174 1
e 175 176 1
e 175 177 1
e 176 177 1
e 178 179 1
e 178 180 1
e 179 180 1
e 181 182 1
e 181 183 1
e 182 183 1
e 184 185 1
e 184 186 1
e 185 186 1
e 187 188 1
e 187 189 1
e 188 189 1
e 190 191 1
e 190 192 1
e 191 192 1
e 193 194 1
e 193 195 1
e 194 195 1
e 196 197 1
e 196 198 1
e 197 198 1
e 199 200 1
e 199 201 1
e 200 201 1
e 202 203 1
e 202 204 1
e 203 204 1
e 205 206 1
e 205 207 1
e 206 207 1
e 208 209 1
e 208 210 1
e 209 210 1
e 211 212 1
e 211 213 1
e 212 213 1
e 214 215 1
e 214 216 1
e 215 216 1
e 217 218 1
e 217 219 1
e 218 219 1
e 220 221 1
e 220 222 1
e 221 222 1
e 223 224 1
e 223 225 1
e 224 225 1
e 226 227 1
e 226 228 1
e 227 228 1
e 229 230 1
e 229 231 1
e 230 231 1
e 232 233 1
e 232 234 1
e 233 234 1
e 235 236 1
e 235 237 1
e 236 237 1
e 238 239 1
e 238 240 1
e 239 240 1
e 241 242 1
e 241 243 1
e 242 243 1
e 244 245 1
e 244 246 1
e 245 246 1
e 247 248 1
e 247 249 1
e 248 249 1
e 250 251 1
e 250 252 1
e 251 252 1
e 253 254 1
e 253 255 1
e 254 255 1
e 256 257 1
e 256 258 1
e 257 258 1
e 259 260 1
e 259 261 1
e 260 261 1
e 262 263 1
e 262 264 1
e 263 264 1
e 265 266 1
e 265 267 1
e 266 267 1
e 268 269 1
e 268 270 1
e 269 270 1
e 271 272 1
e 271 273 1
e 272 273 1
e 274 275 1
e 274 276 1
e 275 276 1
e 277 278 1
e 277 279 1
e 278 279 1
e 280 281 1
e 280 282 1
e 281 282 1
e 283 284 1
e 283 285 1
e 284 285 1
e 286 287 1
e 286 288 1
e 287 288 1
e 289 290 1
e 289 291 1
e 290 291 1
e 292 293 1
e 292 294 1
e 293 294 1
e 295 296 1
e 295 297 1
e 296 297 1
e 298 299 1
e 298 300 1
e 299 300 1
e 301 302 1
e 301 303 1
e 302 303 1
e 304 305 1
e 304 306 1
e 305 306 1
e 307 308 1
e 307 309 1
e 308 309 1
e 310 311 1
e 310 312 1
e 311 312 1
e 313 314 1
e 313 315 1
e 314 315 1
e 316 317 1
e 316 318 1
e 317 318 1
e 319 320 1
e 319 321 1
e 320 321 1
e 322 323 1
e 322 324 1
e 323 324 1
e 325 326 1
e 325 327 1
e 326 327 1
e 328 329 1
e 328 330 1
e 329 330 1
e 331 332 1
e 331 333 1
e 332 333 1
e 334 335 1
e 334 336 1
e 335 336 1
e 337 338 1
e 337 339 1
e 338 339 1
e 340 341 1
e 340 342 1
e 341 342 1
e 343 344 1
e 343 345 1
e 344 345 1
e 346 347 1
e 346 348 1
e 347 348 1
e 349 350 1
e 349 351 1
e 350 351 1
e 352 353 1
e 352 354 1
e 353 354 1
e 355 356 1
e 355 357 1
e 356 357 1
e 358 359 1
e 358 360 1
e 359 360 1
e 361 362 1
e 361 363 1
e 362 363 1
e 364 365 1
e 364 366 1
e 365 366 1
e 367 368 1
e 367 369 1
e 368 369 1
e 370 371 1
e 370 372 1
e 371 372 1
e 373 374 1
e 373 375 1
e 374 375 1
e 376 377 1
e 376 378 1
e 377 378 1
e 379 380 1
e 379 381 1
e 380 381 1
e 382 383 1
e 382 384 1
e 383 384 1
e 385 386 1
e 385 387 1
e 386 387 1
e 388 389 1
e 388 390 1
e 389 390 1
e 391 392 1
e 391 393 1
e 392 393 1
e 394 395 1
e 394 396 1
e 395 396 1
e 397 398 1
e 397 399 1
e 398 399 1
e 400 401 1
e 400 402 1
e 401 402 1
e 403 404 1
e 403 405 1
e 404 405 1
e 406 407 1
e 406 408 1
e 407 408 1
e 409 410 1
e 409 411 1
e 410 411 1
e 412 413 1
e 412 414 1
e 413 414 1
e 415 416 1
e 415 417 1
e 416 417 1
e 418 419 1
e 418 420 1
e 419 420 1
e 421 422 1
e 421 423 1
e 422 423 1
e 424 425 1
e 424 426 1
e 425 426 1
e 427 428 1
e 427 429 1
e 428 429 1
e 430 431 1
e 430 432 1
e 431 432 1
e 433 434 1
e 433 435 1
e 434 435 1
e 436 437 1
e 436 438 1
e 437 438 1
e 439 440 1
e 439 441 1
e 440 441 1
e 442 443 1
e 442 444 1
e 443 444 1
e 445 446 1
e 445 447 1
e 446 447 1
e 448 449 1
e 448 450 1
e 449 450 1
e 451 452 1
e 451 453 1
e 452 453 1
e 454 455 1
e 454 456 1
e 455 456 1
e 457 458 1
e 457 459 1
e 458 459 1
e 460 461 1
e 460 462 1
e 461 462 1
e 463 464 1
e 463 465 1
e 464 465 1
e 466 467 1
e 466 468 1
e 467 468 1
e 469 470 1
e 469 471 1
e 470 471 1
e 472 473 1
e 472 474 1
e 473 474 1
e 475 476 1
e 475 477 1
e 476 477 1
e 478 479 1
e 478 480 1
e 479 480 1
e 481 482 1
e 481 483 1
e 482 483 1
e 484 485 1
e 484 486 1
e 485 486 1
e 487 488 1
e 487 489 1
e 488 489 1
e 490 491 1
e 490 492 1
e 491 492 1
e 493 494 1
e 493 495 1
e 494 495 1
e 496 497 1
e 496 498 1
e 497 498 1
e 499 500 1
e 499 501 1
e 500 501 1
e 502 503 1
e 502 504 1
e 503 504 1
e 505 506 1
e 505 507 1
e 506 507 1
e 508 509 1
e 508 510 1
e 509 510 1
e 511 512 1
e 511 513 1
e 512 513 1
e 514 515 1
e 514 516 1
e 515 516 1
e 517 518 1
e 517 519 1
e 518 519 1
e 520 521 1
e 520 522 1
e 521 522 1
e 523 524 1
e 523 525 1
e 524 525 1
e 526 527 1
e 526 528 1
e 527 528 1
e 529 530 1
e 529 531 1
e 530 531 1
e 532 533 1
e 532 534 1
e 533 534 1
e 535 536 1
e 535 537 1
e 536 537 1
e 538 539 1
e 538 540 1
e 539 540 1
e 541 542 1
e 541 543 1
e 542 543 1
e 544 545 1
e 544 546 1
e 545 546 1
e 547 548 1
e 547 549 1
e 548 549 1
e 550 551 1
e 550 552 1
e 551 552 1
e 553 554 1
e 553 555 1
e 554 555 1
e 556 557 1
e 556 558 1
e 557 558 1
e 559 560 1
e 559 561 1
e 560 561 1
e 562 563 1
e 562 564 1
e 563 564 1
e 565 566 1
e 565 567 1
e 566 567 1
e 568 569 1
e 568 570 1
e 569 570 1
e 571 572 1
e 571 573 1
e 572 573 1
e 574 575 1
e 574 576 1
e 575 576 1
e 577 578 1
e 577 579 1
e 578 579 1
e 580 581 1
e 580 582 1
e 581 582 1
e 583 584 1
e 583 585 1
e 584 585 1
e 586 587 1
e 586 588 1
e 587 588 1
e 589 590 1
e 589 591 1
e 590 591 1
e 592 593 1
e 592 594 1
e 593 594 1
e 595 596 1
e 595 597 1
e 596 597 1
e 598 599 1
e 598 600 1
e 599 600 1
e 601 602 1
e 601 603 1
e 602 603 1
e 604 605 1
e 604 606 1
e 605 606 1
e 607 608 1
e 607 609 1
e 608 609 1
e 610 611 1
e 610 612 1
e 611 612 1
e 613 614 1
e 613 615 1
e 614 615 1
e 616 617 1
e 616 618 1
e 617 618 1
e 619 620 1
e 619 621 1
e 620 621 1
e 622 623 1
e 622 624 1
e 623 624 1
e 625 626 1
e 625 627 1
e 626 627 1
e 628 629 1
e 628 630 1
e 629 630 1
e 631 632 1
e 631 633 1
e 632 633 1
e 634 635 1
e 634 636 1
e 635 636 1
e 637 638 1
e 637 639 1
e 638 639 1
e 640 641 1
e 640 642 1
e 641 642 1
e 643 644 1
e 643 645 1
e 644 645 1
e 646 647 1
e 646 648 1
e 647 648 1
e 649 650 1
e 649 651 1
e 650 651 1
e 652 653 1
e 652 654 1
e 653 654 1
e 655 656 1
e 655 657 1
e 656 657 1
e 658 659 1
e 658 660 1
e 659 660 1
e 661 662 1
e 661 663 1
e 662 663 1
e 664 665 1
e 664 666 1
e 665 666 1
e 667 668 1
e 667 669 1
e 668 669 1
e 670 671 1
e 670 672 1
e 671 672 1
e 673 674 1
e 673 675 1
e 674 675 1
e 676 677 1
e 676 678 1
e 677 678 1
e 679 680 1
e 679 681 1
e 680 681 1
e 682 683 1
e 682 684 1
e 683 684 1
e 685 686 1
e 685 687 1
e 686 687 1
e 688 689 1
e 688 690 1
e 689 690 1
e 691 692 1
e 691 693 1
e 692 693 1
e 694 695 1
e 694 696 1
e 695 696 1
e 697 698 1
e 697 699 1
e 698 699 1
e 700 701 1
e 700 702 1
e 701 702 1
e 703 704 1
e 703 705 1
e 704 705 1
e 706 707 1
e 706 708 1
e 707 708 1
e 709 710 1
e 709 711 1
e 710 711 1
e 712 713 1
e 712 714 1
e 713 714 1
e 715 716 1
e 715 717 1
e 716 717 1
e 718 719 1
e 718 720 1
e 719 720 1
e 721 722 1
e 721 723 1
e 722 723 1
e 724 725 1
e 724 726 1
e 725 726 1
e 727 728 1
e 727 729 1
e 728 729 1
e 730 731 1
e 730 732 1
e 731 732 1
e 733 734 1
e 733 735 1
e 734 735 1
e 736 737 1
e 736 738 1
e 737 738 1
e 739 740 1
e 739 741 1
e 740 741 1
e 742 743 1
e 742 744 1
e 743 744 1
e 745 746 1
e 745 747 1
e 746 747 1
e 748 749 1
e 748 750 1
e 749 750 1
e 751 752 1
e 751 753 1
e 752 753 1
e 754 755 1
e 754 756 1
e 755 756 1
e 757 758 1
e 757 759 1
e 758 759 1
e 760 761 1
e 760 762 1
e 761 762 1
e 763 764 1
e 763 765 1
e 764 765 1
e 766 767 1
e 766 768 1
e 767 768 1
e 769 770 1
e 769 771 1
e 770 771 1
e 772 773 1
e 772 774 1
e 773 774 1
e 775 776 1
e 775 777 1
e 776 777 1
e 778 779 1
e 778 780 1
e 779 780 1
e 781 782 1
e 781 783 1
e 782 783 1
e 784 785 1
e 784 786 1
e 785 786 1
e 787 788 1
e 787 789 1
e 788 789 1
e 790 791 1
e 790 792 1
e 791 792 1
e 793 794 1
e 793 795 1
e 794 795 1
e 796 797 1
e 796 798 1
e 797 798 1
e 799 800 1
e 799 801 1
e 800 801 1
e 802 803 1
e 802 804 1
e 803 804 1
e 805 806 1
e 805 807 1
e 806 807 1
e 808 809 1
e 808 810 1
e 809 810 1
e 811 812 1
e 811 813 1
e 812 813 1
e 814 815 1
e 814 816 1
e 815 816 1
e 817 818 1
e 817 819 1
e 818 819 1
e 820 821 1
e 820 822 1
e 821 822 1
e 823 824 1
e 823 825 1
e 824 825 1
e 826 827 1
e 826 828 1
e 827 828 1
e 829 830 1
e 829 831 1
e 830 831 1
e 832 833 1
e 832 834 1
e 833 834 1
e 835 836 1
e 835 837 1
e 836 837 1
e 838 839 1
e 838 840 1
e 839 840 1
e 841 842 1
e 841 843 1
e 842 843 1
e 844 845 1
e 844 846 1
e 845 846 1
e 847 848 1
e 847 849 1
e 848 849 1
e 850 851 1
e 850 852 1
e 851 852 1
e 853 854 1
e 853 855 1
e 854 855 1
e 856 857 1
e 856 858 1
e 857 858 1
e 859 860 1
e 859 861 1
e 860 861 1
e 862 863 1
e 862 864 1
e 863 864 1
e 865 866 1
e 865 867 1
e 866 867 1
e 868 869 1
e 868 870 1
e 869 870 1
e 871 872 1
e 871 873 1
e 872 873 1
e 874 875 1
e 874 876 1
e 875 876 1
e 877 878 1
e 877 879 1
e 878 879 1
e 880 881 1
e 880 882 1
e 881 882 1
e 883 884 1
e 883 885 1
e 884 885 1
e 886 887 1
e 886 888 1
e 887 888 1
e 889 890 1
e 889 891 1
e 890 891 1
e 892 893 1
e 892 894 1
e 893 894 1
e 895 896 1
e 895 897 1
e 896 897 1
e 898 899 1
e 898 900 1
e 899 900 1
e 901 902 1
e 901 903 1
e 902 903 1
e 904 905 1
e 904 906 1
e 905 906 1
e 907 908 1
e 907 909 1
e 908 909 1
e 910 911 1
e 910 912 1
e 911 912 1
e 913 914 1
e 913 915 1
e 914 915 1
e 916 917 1
e 916 918 1
e 917 918 1
e 919 920 1
e 919 921 1
e 920 921 1
e 922 923 1
e 922 924 1
e 923 924 1
e 925 926 1
e 925 927 1
e 926 927 1
e 928 929 1
e 928 930 1
e 929 930 1
e 931 932 1
e 931 933 1
e 932 933 1
e 934 935 1
e 934 936 1
e 935 936 1
e 937 938 1
e 937 939 1
e 938 939 1
e 940 941 1
e 940 942 1
e 941 942 1
e 943 944 1
e 943 945 1
e 944 945 1
e 946 947 1
e 946 948 1
e 947 948 1
e 949 950 1
e 949 951 1
e 950 951 1
e 952 953 1
e 952 954 1
e 953 954 1
e 955 956 1
e 955 957 1
e 956 957 1
e 958 959 1
e 958 960 1
e 959 960 1
e 961 962 1
e 961 963 1
e 962 963 1
e 964 965 1
e 964 966 1
e 965 966 1
e 967 968 1
e 967 969 1
e 968 969 1
e 970 971 1
e 970 972 1
e 971 972 1
e 973 974 1
e 973 975 1
e 974 975 1
e 976 977 1
e 976 978 1
e 977 978 1
e 979 980 1
e 979 981 1
e 980 981 1
e 982 983 1
e 982 984 1
e 983 984 1
e 985 986 1
e 985 987 1
e 986 987 1
e 988 989 1
e 988 990 1
e 989 990 1
e 991 992 1
e 991 993 1
e 992 993 1
e 994 995 1
e 994 996 1
e 995 996 1
e 997 998 1
e 997 999 1
e 998 999 1
e 1000 1001 1
e 1000 1002 1
e 1001 1002 1
e 1 4 1
e 5 8 1
e 9 12 1
e 10 13 1
e 14 17 1
e 18 21 1
e 19 22 1
e 23 26 1
e 27 30 1
e 28 31 1
e 32 35 1
e 36 39 1
e 37 40 1
e 41 44 1
e 45 48 1
e 46 49 1
e 50 53 1
e 54 57 1
e 55 58 1
e 59 62 1
e 63 66 1
e 64 67 1
e 68 71 1
e 72 75 1
e 73 76 1
e 77 80 1
e 81 84 1
e 82 85 1
e 86 89 1
e 90 93 1
e 91 94 1
e 95 98 1
e 99 102 1
e 100 103 1
e 104 107 1
e 108 111 1
e 109 112 1
e 113 116 1
e 117 120 1
e 118 121 1
e 122 125 1
e 126 129 1
e 127 130 1
e 131 134 1
e 135 138 1
e 136 139 1
e 140 143 1
e 144 147 1
e 145 148 1
e 149 152 1
e 153 156 1
e 154 157 1
e 158 161 1
e 162 165 1
e 163 166 1
e 167 170 1
e 171 174 1
e 172 175 1
e 176 179 1
e 180 183 1
e 181 184 1
e 185 188 1
e 189 192 1
e 190 193 1
e 194 197 1
e 198 201 1
e 199 202 1
e 203 206 1
e 207 210 1
e 208 211 1
e 212 215 1
e 216 219 1
e 217 220 1
e 221 224 1
e 225 228 1
e 226 229 1
e 230 233 1
e 234 237 1
e 235 238 1
e 239 242 1
e 243 246 1
e 244 247 1
e 248 251 1
e 252 255 1
e 253 256 1
e 257 260 1
e 261 264 1
e 262 265 1
e 266 269 1
e 270 273 1
e 271 274 1
e 275 278 1
e 279 282 1
e 280 283 1
e 284 287 1
e 288 291 1
e 289 292 1
e 293 296 1
e 297 300 1
e 298 301 1
e 302 305 1
e 306 309 1
e 307 310 1
e 311 314 1
e 315 318 1
e 316 319 1
e 320 323 1
e 324 327 1
e 325 328 1
e 329 332 1
e 333 336 1
e 334 337 1
e 338 341 1
e 342 345 1
e 343 346 1
e 347 350 1
e 351 354 1
e 352 355 1
e 356 359 1
e 360 363 1
e 361 364 1
e 365 368 1
e 369 372 1
e 370 373 1
e 374 377 1
e 378 381 1
e 379 382 1
e 383 386 1
e 387 390 1
e 388 391 1
e 392 395 1
e 396 399 1
e 397 400 1
e 401 404 1
e 405 408 1
e 406 409 1
e 410 413 1
e 414 417 1
e 415 418 1
e 419 422 1
e 423 426 1
e 424 427 1
e 428 431 1
e 432 435 1
e 433 436 1
e 437 440 1
e 441 444 1
e 442 445 1
e 446 449 1
e 450 453 1
e 451 454 1
e 455 458 1
e 459 462 1
e 460 463 1
e 464 467 1
e 468 471 1
e 469 472 1
e 473 476 1
e 477 480 1
e 478 481 1
e 482 485 1
e 486 489 1
e 487 490 1
e 491 494 1
e 495 498 1
e 496 499 1
e 500 503 1
e 504 507 1
e 505 508 1
e 509 512 1
e 513 516 1
e 514 517 1
e 518 521 1
e 522 525 1
e 523 526 1
e 527 530 1
e 531 534 1
e 532 535 1
e 536 539 1
e 540 543 1
e 541 544 1
e 545 548 1
e 549 552 1
e 550 553 1
e 554 557 1
e 558 561 1
e 559 562 1
e 563 566 1
e 567 570 1
e 568 571 1
e 572 575 1
e 576 579 1
e 577 580 1
e 581 584 1
e 585 588 1
e 586 589 1
e 590 593 1
e 594 597 1
e 595 598 1
e 599 602 1
e 603 606 1
e 604 607 1
e 608 611 1
e 612 615 1
e 613 616 1
e 617 620 1
e 621 624 1
e 622 625 1
e 626 629 1
e 630 633 1
e 631 634 1
e 635 638 1
e 639 642 1
e 640 643 1
e 644 647 1
e 648 651 1
e 649 652 1
e 653 656 1
e 657 660 1
e 658 661 1
e 662 665 1
e 666 669 1
e 667 670 1
e 671 674 1
e 675 678 1
e 676 679 1
e 680 683 1
e 684 687 1
e 685 688 1
e 689 692 1
e 693 696 1
e 694 697 1
e 698 701 1
e 702 705 1
e 703 706 1
e 707 710 1
e 711 714 1
e 712 715 1
e 716 719 1
e 720 723 1
e 721 724 1
e 725 728 1
e 729 732 1
e 730 733 1
e 734 737 1
e 738 741 1
e 739 742 1
e 743 746 1
e 747 750 1
e 748 751 1
e 752 755 1
e 756 759 1
e 757 760 1
e 761 764 1
e 765 768 1
e 766 769 1
e 770 773 1
e 774 777 1
e 775 778 1
e 779 782 1
e 783 786 1
e 784 787 1
e 788 791 1
e 792 795 1
e 793 796 1
e 797 800 1
e 801 804 1
e 802 805 1
e 806 809 1
e 810 813 1
e 811 814 1
e 815 818 1
e 819 822 1
e 820 823 1
e 824 827 1
e 828 831 1
e 829 832 1
e 833 836 1
e 837 840 1
e 838 841 1
e 842 845 1
e 846 849 1
e 847 850 1
e 851 854 1
e 855 858 1
e 856 859 1
e 860 863 1
e 864 867 1
e 865 868 1
e 869 872 1
e 873 876 1
e 874 877 1
e 878 881 1
e 882 885 1
e 883 886 1
e 887 890 1
e 891 894 1
e 892 895 1
e 896 899 1
e 900 903 1
e 901 904 1
e 905 908 1
e 909 912 1
e 910 913 1
e 914 917 1
e 918 921 1
e 919 922 1
e 923 926 1
e 927 930 1
e 928 931 1
e 932 935 1
e 936 939 1
e 937 940 1
e 941 944 1
e 945 948 1
e 946 949 1
e 950 953 1
e 954 957 1
e 955 958 1
e 959 962 1
e 963 966 1
e 964 967 1
e 968 971 1
e 972 975 1
e 973 976 1
e 977 980 1
e 981 984 1
e 982 985 1
e 986 989 1
e 990 993 1
e 991 994 1
e 995 998 1
e 999 1002 1
