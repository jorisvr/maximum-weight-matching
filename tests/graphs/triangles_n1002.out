s 501
m 2 3
m 5 6
m 7 8
m 10 11
m 13 15
m 16 18
m 20 21
m 23 24
m 25 26
m 28 29
m 31 33
m 34 36
m 38 39
m 41 42
m 43 44
m 46 47
m 49 51
m 52 54
m 56 57
m 59 60
m 61 62
m 64 65
m 67 69
m 70 72
m 74 75
m 77 78
m 79 80
m 82 83
m 85 87
m 88 90
m 92 93
m 95 96
m 97 98
m 100 101
m 103 105
m 106 108
m 110 111
m 113 114
m 115 116
m 118 119
m 121 123
m 124 126
m 128 129
m 131 132
m 133 134
m 136 137
m 139 141
m 142 144
m 146 147
m 149 150
m 151 152
m 154 155
m 157 159
m 160 162
m 164 165
m 167 168
m 169 170
m 172 173
m 175 177
m 178 180
m 182 183
m 185 186
m 187 188
m 190 191
m 193 195
m 196 198
m 200 201
m 203 204
m 205 206
m 208 209
m 211 213
m 214 216
m 218 219
m 221 222
m 223 224
m 226 227
m 229 231
m 232 234
m 236 237
m 239 240
m 241 242
m 244 245
m 247 249
m 250 252
m 254 255
m 257 258
m 259 260
m 262 263
m 265 267
m 268 270
m 272 273
m 275 276
m 277 278
m 280 281
m 283 285
m 286 288
m 290 291
m 293 294
m 295 296
m 298 299
m 301 303
m 304 306
m 308 309
m 311 312
m 313 314
m 316 317
m 319 321
m 322 324
m 326 327
m 329 330
m 331 332
m 334 335
m 337 339
m 340 342
m 344 345
m 347 348
m 349 350
m 352 353
m 355 357
m 358 360
m 362 363
m 365 366
m 367 368
m 370 371
m 373 375
m 376 378
m 380 381
m 383 384
m 385 386
m 388 389
m 391 393
m 394 396
m 398 399
m 401 402
m 403 404
m 406 407
m 409 411
m 412 414
m 416 417
m 419 420
m 421 422
m 424 425
m 427 429
m 430 432
m 434 435
m 437 438
m 439 440
m 442 443
m 445 447
m 448 450
m 452 453
m 455 456
m 457 458
m 460 461
m 463 465
m 466 468
m 470 471
m 473 474
m 475 476
m 478 479
m 481 483
m 484 486
m 488 489
m 491 492
m 493 494
m 496 497
m 499 501
m 502 504
m 506 507
m 509 510
m 511 512
m 514 515
m 517 519
m 520 522
m 524 525
m 527 528
m 529 530
m 532 533
m 535 537
m 538 540
m 542 543
m 545 546
m 547 548
m 550 551
m 553 555
m 556 558
m 560 561
m 563 564
m 565 566
m 568 569
m 571 573
m 574 576
m 578 579
m 581 582
m 583 584
m 586 587
m 589 591
m 592 594
m 596 597
m 599 600
m 601 602
m 604 605
m 607 609
m 610 612
m 614 615
m 617 618
m 619 620
m 622 623
m 625 627
m 628 630
m 632 633
m 635 636
m 637 638
m 640 641
m 643 645
m 646 648
m 650 651
m 653 654
m 655 656
m 658 659
m 661 663
m 664 666
m 668 669
m 671 672
m 673 674
m 676 677
m 679 681
m 682 684
m 686 687
m 689 690
m 691 692
m 694 695
m 697 699
m 700 702
m 704 705
m 707 708
m 709 710
m 712 713
m 715 717
m 718 720
m 722 723
m 725 726
m 727 728
m 730 731
m 733 735
m 736 738
m 740 741
m 743 744
m 745 746
m 748 749
m 751 753
m 754 756
m 758 759
m 761 762
m 763 764
m 766 767
m 769 771
m 772 774
m 776 777
m 779 780
m 781 782
m 784 785
m 787 789
m 790 792
m 794 795
m 797 798
m 799 800
m 802 803
m 805 807
m 808 810
m 812 813
m 815 816
m 817 818
m 820 821
m 823 825
m 826 828
m 830 831
m 833 834
m 835 836
m 838 839
m 841 843
m 844 846
m 848 849
m 851 852
m 853 854
m 856 857
m 859 861
m 862 864
m 866 867
m 869 870
m 871 872
m 874 875
m 877 879
m 880 882
m 884 885
m 887 888
m 889 890
m 892 893
m 895 897
m 898 900
m 902 903
m 905 906
m 907 908
m 910 911
m 913 915
m 916 918
m 920 921
m 923 924
m 925 926
m 928 929
m 931 933
m 934 936
m 938 939
m 941 942
m 943 944
m 946 947
m 949 951
m 952 954
m 956 957
m 959 960
m 961 962
m 964 965
m 967 969
m 970 972
m 974 975
m 977 978
m 979 980
m 982 983
m 985 987
m 988 990
m 992 993
m 995 996
m 997 998
m 1000 1001
m 1 4
m 9 12
m 14 17
m 19 22
m 27 30
m 32 35
m 37 40
m 45 48
m 50 53
m 55 58
m 63 66
m 68 71
m 73 76
m 81 84
m 86 89
m 91 94
m 99 102
m 104 107
m 109 112
m 117 120
m 122 125
m 127 130
m 135 138
m 140 143
m 145 148
m 153 156
m 158 161
m 163 166
m 171 174
m 176 179
m 181 184
m 189 192
m 194 197
m 199 202
m 207 210
m 212 215
m 217 220
m 225 228
m 230 233
m 235 238
m 243 246
m 248 251
m 253 256
m 261 264
m 266 269
m 271 274
m 279 282
m 284 287
m 289 292
m 297 300
m 302 305
m 307 310
m 315 318
m 320 323
m 325 328
m 333 336
m 338 341
m 343 346
m 351 354
m 356 359
m 361 364
m 369 372
m 374 377
m 379 382
m 387 390
m 392 395
m 397 400
m 405 408
m 410 413
m 415 418
m 423 426
m 428 431
m 433 436
m 441 444
m 446 449
m 451 454
m 459 462
m 464 467
m 469 472
m 477 480
m 482 485
m 487 490
m 495 498
m 500 503
m 505 508
m 513 516
m 518 521
m 523 526
m 531 534
m 536 539
m 541 544
m 549 552
m 554 557
m 559 562
m 567 570
m 572 575
m 577 580
m 585 588
m 590 593
m 595 598
m 603 606
m 608 611
m 613 616
m 621 624
m 626 629
m 631 634
m 639 642
m 644 647
m 649 652
m 657 660
m 662 665
m 667 670
m 675 678
m 680 683
m 685 688
m 693 696
m 698 701
m 703 706
m 711 714
m 716 719
m 721 724
m 729 732
m 734 737
m 739 742
m 747 750
m 752 755
m 757 760
m 765 768
m 770 773
m 775 778
m 783 786
m 788 791
m 793 796
m 801 804
m 806 809
m 811 814
m 819 822
m 824 827
m 829 832
m 837 840
m 842 845
m 847 850
m 855 858
m 860 863
m 865 868
m 873 876
m 878 881
m 883 886
m 891 894
m 896 899
m 901 904
m 909 912
m 914 917
m 919 922
m 927 930
m 932 935
m 937 940
m 945 948
m 950 953
m 955 958
m 963 966
m 968 971
m 973 976
m 981 984
m 986 989
m 991 994
m 999 1002
